project/profiles/capacity4more/modules/c4m/content/c4m_content_article/c4m_content_article.module

```diff
@@ -146,3 +146,27 @@ function c4m_content_article_c4m_offcanvas_trigger_label_left_alter(&$trigger_la
     $trigger_label['icon'] = 'filter';
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function c4m_content_article_preprocess(&$variables) {
+  if ($variables['theme_hook_original'] != 'page') {
+    return;
+  }
+
+  if ($variables['node']->type != 'article') {
+    return;
+  }
+
+  // Get the tree array, and slice it to start from the second child.
```

Comment:
@nirgn975  ^^
please describe what's on the first child that is not needed the `<i>` tag, and so on ...